storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -1,17 +1,19 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="persistent-storage-using-lvms"]
-= Persistent storage using logical volume manager storage
+= Persistent storage using Logical Volume Manager Storage
 include::_attributes/common-attributes.adoc[]
 :context: logical-volume-manager-storage
 
 toc::[]
 
-{lvms-first} uses the TopoLVM CSI driver to dynamically provision local storage on the clusters.
+Logical Volume Manager (LVM) Storage uses the TopoLVM CSI driver to dynamically provision local storage on the {product-title} clusters.
```

Comment:
```suggestion
Logical Volume Manager Storage uses the TopoLVM CSI driver to dynamically provision local storage on the {product-title} clusters.
```
Changing this to make it consistent with the title.

Reply:
Updated as per the suggestion.